src/components/core/table/table.scss

```
@@ -14,6 +14,10 @@
   );
 }
 
+.ctw-action-hover {
+  background: linear-gradient(90deg, rgba(249, 250, 251, 0) 0%, #f9fafb 4.17%);
```

this should be equivalent if we don't want to mix rgba and hex values
```suggestion
  background: linear-gradient(90deg, #f9fafb00 0%, #f9fafb 4.17%);
```